docs/DeveloperGuide.md

```diff
@@ -1,38 +1,208 @@
 
 
 ## Product scope
 ### Target user profile
-
-{Describe the target user profile}
+* Small to medium-sized business owners operating as retail stores, warehouses, or online shops.
+* Owners seeking a customizable, command-driven system to efficiently manage inventory and optimize workflows can leverage on Inventra.
 
 ### Value proposition
-
-{Describe the value proposition: what problem does it solve?}
+Inventra provides a fast, command-line driven logistics and inventory management system, allowing a single user to do stock tracking, order management, and operational insights, ensuring quick and snappy access to critical business data.
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...       | I want to ...                                     | So that ...                                                                      |
+|--------|----------------|---------------------------------------------------|----------------------------------------------------------------------------------|
+| v1.0   | business owner | add new products to the inventory                 | I can keep track of stock availability.                                          |
+| v1.0   | business owner | delete discontinued products from inventory       | the inventory can be up-to-date on the existing products.                        |
+| v1.0   | business owner | view current inventory list                       | I can efficiently access in-stock products and to handle restocking of products. |
+| v1.0   | business owner | import and export inventory data into spreadsheet | I can streamline inventory updates.                                              |
+| v1.0   | business owner | customize information type related to products    | I can have the flexibility to manage inventory better.                           |
+| v2.0   | business owner | search for products in the inventory              | I can find specific items quickly.                                               |
+| v2.0   | business owner | delete all products in inventory                  | I can handle incorrect creation of inventory data.                               |
+| v2.0   | business owner | update product details                            | I make adjustments to the inventory when required.                               |
 
 ## Non-Functional Requirements
-
-{Give non-functional requirements}
+* The system should work across different operating systems (Windows, Linux, macOS).
+* Data must be persistent across sessions by saving and fetching from/to a CSV file.
 
 ## Glossary
```

Comment:
Glossary is lacking, please double-check on the additional vocabulary that you have introduced, such as "Records".